bridgesync/processor.go

```diff
@@ -580,6 +580,15 @@ func (p *processor) buildClaimsFilterClause(networkIDs []uint32, fromAddress str
 	return ""
 }
 
+// buildTokenMappingsFilterClause builds the WHERE clause for the token_mapping table
+// based on the provided originTokenAddress
+func (p *processor) buildTokenMappingsFilterClause(originTokenAddress string) string {
+	if originTokenAddress != "" && common.IsHexAddress(originTokenAddress) {
```

Comment:
AFAICT the condition can be simplified, WDYT?

```suggestion
	if common.IsHexAddress(originTokenAddress) {
```